addOns/client/src/main/javahelp/org/zaproxy/addon/client/resources/help/contents/client.html

```diff
@@ -37,6 +37,28 @@ <h3>Client Map</h3>
 Any leaf nodes in the Map with a small red 'minus' sign represent URLs which have been found in the DOM but which have not been directly accessed by ZAP.
 <p>
 Selecting a node that has been visited by ZAP will display details about that node in the Client Details tab.
+<p>
+The following right click options are supported:
+
+<h4>Copy URLs to Clipboard</h4>
+
+Copies the URLs of the selected nodes into the clipboard, separated by newlines.
+
+<h4>Delete</h4>
+
+Deletes the selected nodes.
+
+<h4>Open in Browser</h4>
+
+Opens the selected node in the chosen browser.
+This menu item is disabled if multiple nodes are selected.
+If the URL contains a fragment then this will be passed to the browser.
+
+<h4>Show in Sites Tree</h4>
+
+Opens the selected node in the Sites Tree.
```

Comment:
Does this need a mention that the message m the sites tree is the latest and does not necessarily represent the message or action that added it to the client map?